.github/workflows/release.yml

```diff
@@ -0,0 +1,34 @@
+name: Release
+on:
+  push:
+    branches:
+      - main
+      - alpha
```

Comment:
We added `beta` just in case in other repos, so maybe we could add it here, too?

Reply:
Sure 👍🏻